source/block-and-log-streamer.ts

```diff
@@ -126,57 +148,57 @@ export class BlockAndLogStreamer<TBlock extends Block, TLog extends Log> {
 		delete this.logFilters[token];
 	};
 
-
 	public readonly subscribeToOnBlockAdded = (onBlockAdded: (block: TBlock) => void): string => {
 		const uuid = `on block added token ${createUuid()}`;
 		this.onBlockAddedSubscribers[uuid] = onBlockAdded;
 		return uuid;
 	};
 
 	public readonly unsubscribeFromOnBlockAdded = (token: string) => {
-		if (!token.startsWith("on block added token ")) throw new Error(`Expected a block added subscription token.  Actual: ${token}`);
+		if (!token.startsWith("on block added token "))
+			throw new Error(`Expected a block added subscription token.  Actual: ${token}`);
 		delete this.onBlockAddedSubscribers[token];
 	};
 
-
 	public readonly subscribeToOnBlockRemoved = (onBlockRemoved: (block: TBlock) => void): string => {
 		const uuid = `on block removed token ${createUuid()}`;
 		this.onBlockRemovedSubscribers[uuid] = onBlockRemoved;
 		return uuid;
 	};
 
 	public readonly unsubscribeFromOnBlockRemoved = (token: string) => {
-		if (!token.startsWith("on block removed token ")) throw new Error(`Expected a block added subscription token.  Actual: ${token}`);
+		if (!token.startsWith("on block removed token "))
+			throw new Error(`Expected a block added subscription token.  Actual: ${token}`);
 		delete this.onBlockRemovedSubscribers[token];
 	};
 
-
 	public readonly subscribeToOnLogAdded = (onLogAdded: (log: TLog) => void): string => {
 		const uuid = `on log added token ${createUuid()}`;
 		this.onLogAddedSubscribers[uuid] = onLogAdded;
 		return uuid;
 	};
 
 	public readonly unsubscribeFromOnLogAdded = (token: string) => {
-		if (!token.startsWith("on log added token ")) throw new Error(`Expected a log added subscription token.  Actual: ${token}`);
+		if (!token.startsWith("on log added token "))
+			throw new Error(`Expected a log added subscription token.  Actual: ${token}`);
 		delete this.onLogAddedSubscribers[token];
 	};
 
-
 	public readonly subscribeToOnLogRemoved = (onLogRemoved: (log: TLog) => void): string => {
 		const uuid = `on log removed token ${createUuid()}`;
 		this.onLogRemovedSubscribers[uuid] = onLogRemoved;
 		return uuid;
 	};
 
 	public readonly unsubscribeFromOnLogRemoved = (token: string) => {
-		if (!token.startsWith("on log removed token ")) throw new Error(`Expected a log added subscription token.  Actual: ${token}`);
+		if (!token.startsWith("on log removed token "))
+			throw new Error(`Expected a log added subscription token.  Actual: ${token}`);
 		delete this.onLogRemovedSubscribers[token];
 	};
 }
 
 function logAndSwallowWrapper<T>(callback: (arg: T) => void, onError: (error: Error) => void): (arg: T) => void {
-	return function (parameter) {
+	return function(parameter) {
```

Comment:
This should be an arrow function, not sure why it isn't.
```typescript
return parameter => {
    ...
};
```

Reply:
Fixed.